File: content/examples/other/leetcode/problems/contains-duplicate/example-1/solution.js
function containsDuplicate(nums) {
  nums = nums.slice().sort((a,b)=>a-b);
  for (let i=1;i<nums.length;i+=1) {
    if (nums[i]===nums[i-1]) return true;
  }
  return false;
}

if (require.main===module) {
  console.log(containsDuplicate([1,2,3,1]));
}

module.exports = { containsDuplicate };

File: content/examples/other/leetcode/problems/contains-duplicate/example-1/Solution.java
import java.util.*;

public class Solution {
  public static boolean containsDuplicate(int[] nums) {
    Arrays.sort(nums);
    for (int i = 1; i < nums.length; i++) {
      if (nums[i] == nums[i-1]) return true;
    }
    return false;
  }

  public static void main(String[] args) {
    System.out.println(containsDuplicate(new int[]{1,2,3,1}));
  }
}
